### PR TITLE
Logging changes according to new logging framework 

### PR DESCRIPTION
**Pull request by anmolsachan**
tendrl-bug-id: Tendrl/specifications#28
tendrl-spec: Tendrl/specifications#127
reference-spec: Tendrl/specifications#94
reference-code-pr: Tendrl/commons#161

Signed-off-by: anmolsachan asachan@redhat.com

### CODE REVIEW

**Comment by anmolsachan** [comment]
@GowthamShanmugam @shtripat please review

File: tendrl/node_agent/discovery/platform/manager.py [hunk 1]
```diff
@@ -30,8 +30,16 @@ def load_plugins(self):
...
+            Event(
+                Message(
+                    priority="error",
+                    publisher=tendrl_ns.publisher_id,
+                    payload={"message": 'Failed to load the platform plugins. '
+                                        'Error %s %s' %
+                                        (ex, traceback.format_exc())
+                             }
+                )
+            )
```
Comment (GowthamShanmugam) [inline tendrl/node_agent/discovery/platform/manager.py]: For this you can use exception message class but it is not merged if it is merged then change as per that

File: tendrl/node_agent/node_sync/disk_sync.py [hunk 2]
```diff
@@ -79,9 +78,21 @@ def get_node_disks():
...
                             rv['used_disks_id'].append(disk['disk_id'])
         else:
-            LOG.error(err)
+            Event(
+                Message(
+                    priority="error",
+                    publisher=tendrl_ns.publisher_id,
+                    payload={"message": err}
+                )
+            )
```
Comment (GowthamShanmugam) [inline tendrl/node_agent/node_sync/disk_sync.py]: Make sure this is 'err' is string, other wise it gives  serialization problem, If Exception message class came then you can use exception as it is. But now you have to use string only.

File: tendrl/node_agent/node_sync/disk_sync.py [hunk 3]
```diff
@@ -79,9 +78,21 @@ def get_node_disks():
...
     else:
-        LOG.error(err)
+        Event(
+            Message(
+                priority="error",
+                publisher=tendrl_ns.publisher_id,
+                payload={"message": err}
+            )
+        )
```
Comment (GowthamShanmugam) [inline tendrl/node_agent/node_sync/disk_sync.py]: In exception class you can use exception object, but now you can't use exception object it gives serialization problem so make sure this "ex" is string or not

File: tendrl/node_agent/discovery/platform/manager.py (hunk 1 above)
Comment (anmolsachan) [inline tendrl/node_agent/discovery/platform/manager.py]: Yeah. I will change this later

File: tendrl/node_agent/node_sync/disk_sync.py (hunk 2 above)
Comment (anmolsachan) [inline tendrl/node_agent/node_sync/disk_sync.py]: Made the changes

File: tendrl/node_agent/node_sync/disk_sync.py (hunk 3 above)
Comment (anmolsachan) [inline tendrl/node_agent/node_sync/disk_sync.py]: Made the changes

**Pull request closed without merging**